python/ray/experimental/raysort/main.py

```diff
@@ -30,25 +34,43 @@ def get_args():
     )
     parser.add_argument(
         "--total_data_size",
-        default=1_000_000_000,
+        default=512 * 1000 * 1024 * 1024,
         type=ByteCount,
         help="partition size in bytes",
     )
     parser.add_argument(
         "--num_mappers",
-        default=4,
+        default=128,
         type=int,
         help="number of map tasks",
     )
     parser.add_argument(
         "--num_reducers",
+        default=16,
+        type=int,
+        help="number of reduce actors",
```

Comment:
Can you clarify that this is the number of second-stage reducers?


It would be helpful to see a description of the design in terms of rounds. Something like this:

M = total # of map tasks
R = total # of rounds
N = total # of nodes

round = 1 set of map tasks which will get merged together. The second-stage reducers merge the results of each round.